apps/src/dance/ai/DanceAiModal.tsx

```diff
@@ -495,6 +495,7 @@ const DanceAiModal: React.FunctionComponent = () => {
     return (
       <div
         className={moduleStyles.inputsContainer}
+        tabIndex={0}
```

Comment:
Does this also need an `aria-label` if it's tab-navigable?

Reply:
"emoji, emoji, emoji" is probably fine, but clicking this also has the action of start over so I was thinking an `aria-label` of "Start Over" maybe? Might be good to double check with @samantha-code

Reply:
Agreed with @Sam-Hutchins that we should leave it "emoji, emoji, emoji" since we'll make this group not clickable a little bit later anyways

Reply:
Sounds good, should we go ahead and remove the `tabIndex` then as well?

Reply:
I think we should keep it for now and remove it later together with onClick to make it more consistent. (tabNavigable if clickable, not tabNavigable if not clickable)